src/hooks/useNetInfo.ts

```diff
@@ -0,0 +1,14 @@
+import NetInfo from '@react-native-community/netinfo';
+import {useEffect, useState} from 'react';
+
+export function useNetInfo() {
+  const [networkStatus, setNetworkStatus] = useState<boolean>(false);
+
+  useEffect(() => {
+    NetInfo.addEventListener((state) => {
+      setNetworkStatus(state.isConnected ? true : false);
```

Comment:
NIT 💅 
```suggestion
      setNetworkStatus(state.isConnected);
```

Reply:
I can be null, so I added this condition to explicitly set to true or false.

Reply:
I see, it can also be `Boolean(state.isConnected)`, But it's fine too.